File: typescript/iniciante/1010.ts
import { bold, cyan } from "./../fmt/colors";

const input1 = prompt("Peça 1 (código quantidade preço):")?.split(" ") ?? [];
const itemCode1 = Number(input1[0]);
const itemQuantity1 = Number(input1[1]);
const itemUnitPrice1 = Number(input1[2]?.replace(",", "."));

const input2 = prompt("Peça 2 (código quantidade preço):")?.split(" ") ?? [];
const itemCode2 = Number(input2[0]);
const itemQuantity2 = Number(input2[1]);
const itemUnitPrice2 = Number(input2[2]?.replace(",", "."));

const totalValue =
  itemQuantity1 * itemUnitPrice1 + itemQuantity2 * itemUnitPrice2;

console.log(cyan(`VALOR A PAGAR: R$ ${bold(totalValue.toFixed(2))}`));

File: swift/iniciante/1010.swift
import Foundation

let input1 = readLine()!.split(separator: " ").map { String($0) }
let itemCode1 = Int(input1[0])!
let itemQuantity1 = Int(input1[1])!
let itemUnitPrice1 = Double(input1[2].replacingOccurrences(of: ",", with: "."))!

let input2 = readLine()!.split(separator: " ").map { String($0) }
let itemCode2 = Int(input2[0])!
let itemQuantity2 = Int(input2[1])!
let itemUnitPrice2 = Double(input2[2].replacingOccurrences(of: ",", with: "."))!

let totalValue = Double(itemQuantity1) * itemUnitPrice1 + Double(itemQuantity2) * itemUnitPrice2

print(String(format: "VALOR A PAGAR: R$ %.2f", totalValue))
